src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -58,7 +58,8 @@ protected function describeRoute(Route $route, array $options = array())
             ."\n".'- Class: '.\get_class($route)
             ."\n".'- Defaults: '.$this->formatRouterConfig($route->getDefaults())
             ."\n".'- Requirements: '.($route->getRequirements() ? $this->formatRouterConfig($route->getRequirements()) : 'NO CUSTOM')
-            ."\n".'- Options: '.$this->formatRouterConfig($route->getOptions());
+            ."\n".'- Options: '.$this->formatRouterConfig($route->getOptions())
+            ."\n".'- Condition: '.('' !== $route->getCondition() ? $route->getCondition() : 'NO CONDITION');
```

Comment:
i'd prefer the empty string here, like `Host Regex`

Reply:
ok 👍